kubernetes/mysql-trillian/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM mysql:5.7
```

Comment:
What do you think about moving these files to 
`deploy/mysql-trillian`
`deploy/kubernetes`

?